### PR TITLE
Autostart the service

### PR DESCRIPTION
**Pull request by rubo77**

### CODE REVIEW

File: ff-node-monitor.service [hunk 1]
```diff
@@ -8,6 +8,7 @@ Environment=ROCKET_ENV=production
 WorkingDirectory=/var/lib/ff-node-monitor/src
 ExecStart=/var/lib/ff-node-monitor/src/target/release/ff-node-monitor
 User=ff-node-monitor
+Restart=always
```
Comment (RalfJung) [inline ff-node-monitor.service]: I don't think `always` is the setting we want, can you change this to `on-failure`?

Comment (rubo77) [reply ff-node-monitor.service]: What about after a reboot?

I think always is best

Comment (RalfJung) [reply ff-node-monitor.service]: This is not at all about reboots. The difference between `always` and `on-failure` is about what happens when the service exist with status code 0.

The man page for this option explicitly says

> Setting this to on-failure is the recommended choice for long-running services, in order to increase reliability by attempting automatic recovery from errors.

Comment (rubo77) [reply ff-node-monitor.service]: Will it start then automatically after a reboot?

Comment (RalfJung) [reply ff-node-monitor.service]: Yes. `Restart` has nothing to do with system reboots, that is entirely controlled by `systemctl enable`.

`Restart` is about what happens when the service is running, and then exits. I suggest you read the corresponding section of `man systemd.service`.

**Comment by RalfJung** [comment]
I'll merge this and do the remaining fixes myself.

**Pull request merged**